recipes/melodies-monet/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "melodies-monet" %}
+{% set version = "0.1.dev1" %}
```

Comment:
This one falls under https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Reply:
If I'm reading that correctly, it just means that the following needs to be used in order for the install to work (until there's a stable release)?
```
conda install -c conda-forge/label/melodies-monet_dev -c conda-forge melodies-monet
```

Reply:
That is for the user. You would need to add a `conda_build_config.yaml` here according to https://github.com/conda-forge/cfep/blob/main/cfep-05.md#specification.

If you can request a stable release upstream, you don't need that step.